hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -65,9 +65,17 @@ static qint64 setTime;
 #define QPA_HWC_TIMING_SAMPLE(variable)
 #endif
 
+
+class HwComposerContent_v11;
+
+static int g_external_connected = 0;
+static int g_external_connected_next = 0;
+static int g_unblanked_displays[HWC_NUM_DISPLAY_TYPES] = { 0 };
```

Comment:
Non-CamelCase function names get_screen_size and using of fprintf instead of qDebug irks me slightly, but these platform plugin codes are bit in the middle of two words so maybe mixing C-style with C++-style is fine